MIGRATING.md

```diff
@@ -0,0 +1,153 @@
+
+Because the SignalFx Tracing Library for NodeJS uses OpenTracing and the Splunk Distribution
+of OpenTelemetry for NodeJS uses OpenTelemetry, the semantic
+conventions for span tag names change when you migrate. For more information,
+see [Migrate from OpenTracing to OpenTelemetry](https://docs.signalfx.com/en/latest/apm/apm-getting-started/apm-opentelemetry-collector.html#apm-opentelemetry-migration).
+
+## Getting help
+
+If you experience any issues following the guide below, or something is unclear, or missing, please don't hesitate to
+open an issue in Github. Any and all ideas for improvements are also welcome.
+
+<a name="known-limitations"></a>
+## Known limitations as compared to SignalFx Tracing Library
+
+- Lowest supported version of NodeJS is `v8.5`, [see more information here](https://github.com/open-telemetry/opentelemetry-js#node-support)
+- No auto-instrumentation for:
+  - `AdonisJS`
+  - `amqp10`
+  - `mongodb-core` ([because it's deprecated](https://github.com/mongodb-js/mongodb-core))
+  - `sails`
+- Limited instrumentation for:
+  - `nest` - only manual insturmentation helpers, provided by community
+- other notes on instrumentation:
+  - `express` instrumentation requires `http`/`https` instrumentation
+  - `bluebird` - context propagation only (via `async_hooks`)
+  - `q` - context propagation only (via `async_hooks`)
+  - `when` - context propagation only (via `async_hooks`)
+  - `socket.io` - provided by community (<https://github.com/aspecto-io/opentelemetry-ext-js/tree/master/packages/instrumentation-socket.io>)
+
+## Changes to defaults
+
+- Default flush interval, which defines how frequently captured telemetry data is sent to the backend, is now 30s instead of 2s
+
+## Requirements
+
+This Splunk Distribution of OpenTelemetry requires Node.js 8.5 or later.
+If you're still using an earlier version of Node.js, continue using the SignalFx Tracing Library for Node.js.
+
+Current effective required Node.js version is: ![node-current](https://img.shields.io/node/v/@splunk/otel?style=flat-square)
+
+## Migration steps
+
+### Instrumented libraries
+
+With the exception of [explicitly listed limitations](#known-limitations) we aim to support all libraries supported by
+signalfx-nodejs-tracing. To find an equivalent auto-instrumentation open <https://opentelemetry.io/registry/> and for
+each instrumentation
+[from `signalfx-nodejs-tracing`'s README](https://github.com/signalfx/signalfx-nodejs-tracing/#requirements-and-supported-software)
+search by the name of the library in the registry.
+
+For example, if you'd like to migrate instrumentation for `mysql`, go to
+[https://opentelemetry.io/registry/?s=**mysql**&component=&language=**js**#](https://opentelemetry.io/registry/?s=mysql&component=&language=js#).
+
+Once you have identified an instrumentation package, install it using `npm` or `yarn`.
+
+- If the package is [on this list](./README.md#default-instrumentation-packages-), it
+  will be enabled automatically.
```

Comment:
but not installed automatically, right? I'd stress that somehow or make that more clear unless you think I'm the only one that had to stop here and think.